keps/sig-node/3857-rro-mounts/kep.yaml

```diff
@@ -6,7 +6,7 @@ owning-sig: sig-node
 #participating-sigs:
 #  - sig-aaa
 #  - sig-bbb
-status: implementable
+status: implemented
```

Comment:
I think this can come after we merge the code for stable.

Reply:
Fixed